ArithmeticTools/Rational.swift

```diff
@@ -100,16 +100,15 @@ public protocol Rational:
     init(_ numerator: Int, _ denominator: Int)
 
     /// - returns: Representation of a `Rational` value with a given `numerator`, if possible.
-    /// Otherwise, `nil`.
     ///
     /// > Preserves the arithmetic value of the original `Rational` value.
-    func respelling(numerator: Int) -> Self?
+    func respelling(numerator: Int) -> Self
 
     /// - returns: Representation of a `Rational` value with a given `denominator`, if
```

Comment:
In the Swift stdlib, they use capitalized versions of these labels. I've started doing that. Maybe we normalize these as we see them. The lowercased versions still work for the doc gen, but I think we can adopt the standard.